site/cypress/e2e/Cypress-TAGrading/notebook_section.spec.js

```diff
@@ -0,0 +1,31 @@
+describe('Notebook Section Testing', () => {
+    // Ignore uncaught js exceptions
+    Cypress.on('uncaught:exception', () => {
+        return false;
+    });
+    it('Notebook builder and Notebook panel', () => {
```

Comment:
Can you leave some comments explaining what each section of the test is supposed to test?

Reply:
Done [fcc6046](https://github.com/Submitty/Submitty/pull/10766/commits/fcc6046db5f0a273a8881bb7addc2d1dfeb1bd5a)